tests/Support/Queries/PostQuery.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rebing\GraphQL\Tests\Support\Queries;
+
+use GraphQL\Type\Definition\Type;
+use Rebing\GraphQL\Support\Query;
+use Rebing\GraphQL\Support\SelectFields;
+use Rebing\GraphQL\Support\Facades\GraphQL;
+use Rebing\GraphQL\Tests\Support\Models\Post;
+
+class PostQuery extends Query
+{
+    protected $attributes = [
+        'name' => 'post',
+    ];
+
+    public function type()
+    {
+        return GraphQL::type('Post');
+    }
+
+    public function args()
+    {
+        return [
+            'id' => [
+                'type' => Type::nonNull(Type::id()),
+            ],
+        ];
+    }
+
+    public function resolve($root, $args, SelectFields $selectFields)
+    {
+        return Post
+            ::findOrFail($args['id']);
```

Comment:
Is this kind of formatting standard?

Reply:
Ops, probably not. I'm so used to it because _usually_ I've more conditions and then I start if off like this to get to this:
```php
return Post
  ::select()
  ->where()
  ->where()
  ->orderBy()
  ->get();
```
But yes, pointless here!

Reply:
Fixed (one line)